packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -218,7 +218,11 @@ class Auth {
                 siteId: config.siteId
             },
             throwOnBadResponse: true,
-            fetchOptions: config.fetchOptions
+            // We need to set credentials to 'same-origin' to allow cookies to be set. This is required as SLAS calls return a dwsid cookie for hybrid sites.
+            fetchOptions: {
```

Comment:
May be we should pass it as another prop (slasFetchOptions) to CommerceAPIProvider so that is more generic. Any thoughts?

Reply:
the decision is based on the answer to the question: do we want users to override this option?

Reply:
for hybrid sites NO. For non hybrid sites we don't really care